src/Draco.Compiler/Syntax/Lexer.cs

```diff
@@ -374,6 +374,28 @@ private IToken LexString()
                 }
             }
             offset += mode.ExtendedDelims + 1;
+            // Line continuation           
+            ch = this.Peek(offset);
+            if (char.IsWhiteSpace(ch))
+            {
+                var whiteCharOffset = 0;
+                while (char.IsWhiteSpace(ch) && ch != '\n')
+                {
+                    whiteCharOffset++;
+                    ch = this.Peek(offset + whiteCharOffset);
+                }
+                if (this.TryParseNewline(offset + whiteCharOffset, out int length))
+                {
+                    //TODO: decide what to do if there are whitespaces after line continuation
+                    offset += whiteCharOffset + length;
+                    goto start;
+                }
+                else
+                {
+                    // TODO: Error, there is not newline directly after line continuation
+                    throw new NotImplementedException();
```

Comment:
Is this case surely an error? Does a whitespace after `\` inherently assume line continuation? `\<space>` _is_ an error indeed, but I don't think it's because it was meant to be a line continuation.

Reply:
Well yes but where else do you want to throw? should i just forward it into `ParseEscapeSequence`?

Reply:
My gut feeling would be exactly this, yes. It will just go on to fail in parsing a generic escape sequence